source/agora/crypto/ECC.d

```diff
@@ -129,17 +129,34 @@ public struct Scalar
 
     ***************************************************************************/
 
+    public void toString (scope void delegate(scope const(char)[]) @safe sink)
+        const
+    {
+        FormatSpec!char spec;
+        this.toString(sink, spec);
+    }
+
+    /// Ditto
     public void toString (scope void delegate(scope const(char)[]) @safe sink,
-                          PrintMode mode = PrintMode.Obfuscated) const
+                          scope const ref FormatSpec!char spec) const
     {
-        final switch (mode)
+        switch (spec.spec)
         {
-        case PrintMode.Obfuscated:
+        // Default to obfuscated print mode
+        case 's':
+        default:
             formattedWrite(sink, "**SCALAR**");
             break;
 
-        case PrintMode.Clear:
-            formattedWrite(sink, "%s", this.data);
+        // Clear text was explicitly requested
+        case 'c':
+            this.data.toString(sink);
+            break;
+
+        // Modes supported by BitBlob
+        case 'x':
```

Comment:
Don't we need to add a test for this case?

Reply:
The test is a L190

Reply:
I understand what the code means. Ty.